e2e/_suites/fleet/stand-alone.go

```diff
@@ -75,7 +75,8 @@ func (sats *StandAloneTestSuite) aStandaloneAgentIsDeployed(image string) error
 
 		dockerInstaller.PreInstallFn()
 
-		dockerImageTag += "-amd64"
+		arch := e2e.GetArchitecture()
+		dockerImageTag += "-" + arch
```

Comment:
Docker images are pushed to the docker registry, and their tag name contains the architecture, for further details refers to https://github.com/elastic/beats/pull/24121